static/app/components/accordion/accordion.tsx

```diff
@@ -0,0 +1,84 @@
+import React, {useState} from 'react';
+import styled from '@emotion/styled';
+import {motion} from 'framer-motion';
+
+import {AccordionSummaryProps} from 'sentry/components/accordion/accordionSummary';
+import space from 'sentry/styles/space';
+
+interface AccordionProps extends React.HTMLAttributes<HTMLDivElement> {
+  /**
+   * The body of the accordion that is shown & hidden
+   */
+  accordionDetails: React.ReactNode;
+  /**
+   * The header of the accordion that is always shown
+   */
+  accordionSummary: (p: AccordionSummaryProps) => React.ReactNode;
+  /**
+   * Whether the accordion is disabled (cannot be opened)
+   */
+  disabled?: boolean;
+}
+
+function Accordion({
+  disabled = false,
+  accordionDetails,
+  accordionSummary,
+}: AccordionProps) {
+  const [isExpanded, setIsExpanded] = useState(false);
```

Comment:
So this is an accordion row rather than the whole accordion right?

Can we call it that then? `AccordionRow`